data/systems/custom/07_1orionis.lua

```diff
@@ -9,19 +9,19 @@ local s = CustomSystem:new('1 Orionis',{ 'STAR_F' }) --Alt name will be "Cluaran
 	:govtype('CISLIBDEM')
 	:lawlessness(f(5,100)) -- 1/100th from a peaceful eden
 	:short_desc('A proudly independent hisoric colony')
-	:long_desc([["Proud World for Proud People". 
+	:long_desc([["Proud World for Proud People".
 1 Orionis' motto can be traced back to year 2325, as the "Evergreen" -one of the legacy colony ships launched before the era of hyperspace travel- landed on planet Saorsa. Its founding fathers vowed to never live again under Terran oppression,
 This heritage still shapes the system’s values to this day : independence, pride, and freedom.
 1 Orionis acively took part in the War of Hope (2723-2725), as its volunteer forces fought alongside Epsilon Eridani against Sol’s attempt of hegemony. The system is a founding member of the Commonwealth of Independent Worlds and remains politically influent to this day.
-In present day, the systems boasts a strong industry and is the birthplace of many an elite pilot of the Confederation Fleet. 
+In present day, the systems boasts a strong industry and is the birthplace of many an elite pilot of the Confederation Fleet.
 Travellers will also find the citizens are a proud lot : be it of their tradition, their chilly climate, their language, their gas giants...]])
```

Comment:
While you're at it, there are a few issues with the long description (missing 'the', em-dashes formatted as hyphens, trailing commas instead of periods, incorrect spacing around colons, "politically influent" -> "influential", "the systems", etc.) that I'd like to see fixed.

Reply:
What's the policy for quotes - straight or typographical?

Reply:
At your discretion - for system definitions typographical quotes are definitely an option, though it's more a matter of how much effort you'd like to put in.